src/adapters/mc/sshpreview/refresh_for_preview.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -n "$(command -v yum)" ]; then
+        echo "using yum"
+        yum remove openssh-server -y
+        rm /etc/ssh/sshd_config*
+        rm /etc/ssh/sshd_config.d/osconfig_*
+        yum install openssh-server -y
+
+        if [[ $(yum list installed | grep -wi osconfig) ]]; then
+                echo "OSConfig Installed, updating"
+                yum update
+                yum install osconfig
+        else
+                echo "OSConfig Not Present"
+        fi
+fi
+
+if [ -n "$(command -v apt-get)" ]; then
+        echo "using apt-get"
+        apt-get remove --purge openssh-server -y
+        rm /etc/ssh/sshd_config*
+        rm /etc/ssh/sshd_config.d/osconfig_*
+        apt-get install openssh-server -y
```

Comment:
Can we add sudo in here to allow the user to not have to run the script elevated?

Reply:
added sudo